components/portfolio/MarketPositions.tsx

```diff
@@ -78,17 +82,33 @@ export const MarketPositions = ({
           },
         ]}
         data={positions.map<TableData>(
-          ({ assetId, price, userBalance, outcome, changePercentage }) => {
+          ({
+            assetId,
+            price,
+            userBalance,
+            outcome,
+            changePercentage,
+            market,
+          }) => {
+            const baseAssetId = parseAssetId(market.baseAsset).unrightOr(null);
```

Comment:
those 4 lines e.g. can they also be replaced with `lookupBaseAssetPrice` function?